config/config-example.js

```diff
@@ -160,6 +160,15 @@ exports.customavatars = {
 	//'userid': 'customavatar.png'
 };
 
+// Tournament announcements
+// When tournaments are created in rooms listed below, they will be announced in
+// the server's main tournament room (either the specified tourroom or by default
+// the room 'tournaments')
+exports.tourroom = '';
+exports.tourannouncements = {
+	//'roomid': 1
```

Comment:
This should be an array. The `:1` micro-optimization stuff shouldn't be exposed to config.